server/src/main/java/org/elasticsearch/rest/action/cat/RestPluginsAction.java

```diff
@@ -106,12 +113,16 @@ private Table buildTable(RestRequest req, ClusterStateResponse state, NodesInfoR
                 continue;
             }
             for (PluginInfo pluginInfo : plugins.getPluginInfos()) {
+                if (pluginInfo.getType() == PluginType.BOOTSTRAP && includeBootstrapPlugins == false) {
```

Comment:
We don't seem to have any unit tests for this class. Would it be possible to add a simple one? The packaging tests added seem to be testing this new flag, but that doesn't seem like the right place, since the flag doesn't directly have anything to do with the quota aware fs plugin, it's just the only example we have so far a bootstrap plugin.

Reply:
@rjernst I've added unit tests, please take a look.